docs/Common/Enums/index.md

```diff
@@ -230,6 +230,17 @@
 |CID1SET|2||
 |CID2SET|3||
 
+### DeactivateReason
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|AUDIO|0||
```

Comment:
I don't believe this `DeactivateReason` is related to status light values?

Reply:
@JackLivio, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/107/commits/13acbe8d0475d92bbd431d2bf48dce0cb53ea485